flash_patcher/parse/visitor/asset_visitor.py

```diff
@@ -22,7 +22,10 @@ def __init__(self: AssetPackProcessor, asset_folder: Path, decomp_location: Path
         self.decomp_location = decomp_location
         self.modified_scripts = set()
 
-    def visitAddAssetBlock(self, ctx: AssetPackParser.AddAssetBlockContext) -> None:
+    def visitAddAssetBlock(
```

Comment:
```suggestion
    def visit_add_asset_block(
```
Styling: Function names should be in `snake_case`. See [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names) for details. That said, is this caused by antlr? I'm not sure on how that works.

Reply:
Yes, this is an ANTLR issue. We're overriding the `visitAddAssetBlock` in the default `PatchfileVisitor` class that ANTLR generates, so there's not much we can do about this.